code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -1,6 +1,30 @@
 /obj/item/weapon/gun/magic/staff/
 	slot_flags = SLOT_BACK
 
+/obj/item/weapon/gun/magic/staff/death
+	name = "staff of death"
+	desc = "An artefact that spits deadly bolts of energy which overwhelm a victim's body with pure energy, slaying them without fail"
+	fire_sound = "sound/magic/WandoDeath.ogg"
+	ammo_type = /obj/item/ammo_casing/magic/death
+	icon_state = "scythe0"
```

Comment:
Trust me, I tried, couldn't find it anywhere. Not even in the default weaponry.dm file. Thanks though, I'll add it.